test/parallel/test-cluster-bind-privileged-port.js

```diff
@@ -27,9 +27,15 @@ const net = require('net');
 const { readFileSync } = require('fs');
 
 if (common.isLinux) {
-  const unprivilegedPortStart = parseInt(readFileSync('/proc/sys/net/ipv4/ip_unprivileged_port_start'));
-  if (unprivilegedPortStart <= 42) {
-    common.skip('Port 42 is unprivileged');
+  try {
+    const sysctlOutput = execSync('sysctl net.ipv4.ip_unprivileged_port_start').toString();
```

Comment:
This basically reverts https://github.com/nodejs/node/commit/f69e84c75f8fbf48e5019af6d8b59e949c5cd653. Can you simply handle the error thrown by `readFileSync('/proc/sys/net/ipv4/ip_unprivileged_port_start')` when the file does not exist?

Reply:
Oh, that is so embarrassing. I am doing this in v19 and v18 (the one I'm actually trying to deploy). Went back to the old one. Ignore this. Will fix in next few hours. ARGGHHH

Reply:
OK, fixed. Needed more coffee. Working in 3 places (v19 source, v18 source, my build).